test/test_YoutubeDL.py

```diff
@@ -874,42 +874,43 @@ def test_format_note(self):
         }), r'^30fps$')
 
     def test_postprocessors(self):
-        filename = 'post-processor-testfile.mp4'
-        audiofile = filename + '.mp3'
+        filename = 'post-processor-testfile'
+        video_file = filename + '.mp4'
+        audio_file = filename + '.mp3'
 
         class SimplePP(PostProcessor):
             def run(self, info):
-                with open(audiofile, 'w') as f:
+                with open(audio_file, 'w') as f:
                     f.write('EXAMPLE')
                 return [info['filepath']], info
 
         def run_pp(params, PP):
-            with open(filename, 'w') as f:
+            with open(video_file, 'w') as f:
                 f.write('EXAMPLE')
             ydl = YoutubeDL(params)
             ydl.add_post_processor(PP())
-            ydl.post_process(filename, {'filepath': filename})
+            ydl.post_process(video_file, {'filepath': video_file})
 
-        run_pp({'keepvideo': True}, SimplePP)
-        self.assertTrue(os.path.exists(filename), '%s doesn\'t exist' % filename)
-        self.assertTrue(os.path.exists(audiofile), '%s doesn\'t exist' % audiofile)
-        os.unlink(filename)
-        os.unlink(audiofile)
+        run_pp({'keepvideo': True, 'outtmpl': filename}, SimplePP)
```

Comment:
Calling this out specifically - these changes necessitated an `outtmpl` (or setting `title` and `id` for the default `outtmpl` to use). I don't think this is an issue, but I wanted to confirm!

Reply:
To expand, the reason is that the old implementation didn't concern itself with outtmpls at all - it just got a list of source/destination filenames and it moved files based on that.

The new implementation uses `prepare_filename` which allows it to keep source/destination logic contained entirely within the `MoveFilesPP` module, but the side effect is that it needs to be able to generate a valid `outtmpl` _somehow_